### DISCUSSION TITLE
[Feature Request] Auto-imports

### DISCUSSION THREAD

**Discussion by transcental** [community/community#15892] [Copilot Conversations]
When I accept a Copilot suggestion that uses functions defined in another file, I would like it to automatically import these. Normally, I would just use native autocomplete to auto-import these, but as Copilot takes priority it doesn't import the functions. This then means I need to go back and import them myself. Or go back and use Ctrl + Space in VSC.

Please could you make it so that if Copilot is using a function from another file it imports it at the top of the currently opened file?

If you want this feature as well, upvote this comment to try and get it to GH's attention.

**Comment by TurtIeSocks** [comment]
Just dropping in to echo this feeling. Bit annoying to fight between VSC intellisense and copilot when trying to auto import functions.

**Comment by NicolasPL64** [comment]
Still waiting for this. So annoying
